joomla/joomla-cms
# [4.0] Fixed menu params for front page item on installation

Pull Request for Issue #32493. Note: This is not an alternative to #32547 - both are necessary. #32547 for fixing the notice and another error, and this PR here for fixing the underlying cause.

### Summary of Changes

The "home" menu item that is created on installation contains parameters from 3.x, and lacks parameters from 4.x. This PR fixes the installation SQL.

### Testing Instructions

IMHO, this could be merged on code review. Testing instructions are a bit more complicated here.

1. Install a clean 4.0-beta7 or nightly.
2. **Important:** Don't edit the "home" menu item!
3. Create 4 articles and set them as featured and published.
4. Look at the home page. The articles should be displayed in a one-column layout.
5. Update using a 4.0-beta7 or nightly update package.
6. Look again at the home page.

Using the prebuilt packages for this PR, do a clean new installation and repeat steps 2-6.

### Actual result BEFORE applying this Pull Request

Before the update, the "home" item displays articles in a one-column layout. After running an update, it displays them in three columns.

### Expected result AFTER applying this Pull Request

Updating should not change the layout.

### Documentation Changes Required

none
